src/SupervisionSuccess/utils/convertCSVStringToParams.js

```diff
@@ -0,0 +1,58 @@
+import csv from "csvtojson";
+
+/**
+ * Functions that extracts needed fields from csv and transforms to params object
```

Comment:
Let's reword this slightly: "Function that extracts needed fields from the given CSV string and transforms them into parameters ready to be fed into the revocations model."